src/Header/ContentType.php

```diff
@@ -238,7 +239,7 @@ protected function assembleValue()
     {
         $mediaType = $this->getMediaType();
         if (empty($this->parameters)) {
-            return $mediaType;
+            return $mediaType ?: '';
```

Comment:
Since the type is `null|string`, should this be `??`?

Reply:
No, because we're still targeting PHP 5.6 with the next minor release.